kubernetes-model-generator/generateModel.sh

```diff
@@ -53,8 +54,40 @@ declare -a modules=(
     "openshift-model-whereabouts"
     "openshift-model-storageversionmigrator"
 )
+
+declare -a extensionModuleParents=(
+    "../extensions/knative/pom.xml"
+    "../extensions/camel-k/pom.xml"
+    "../extensions/camel-k/pom.xml"
+    "../extensions/chaosmesh/pom.xml"
+    "../extensions/service-catalog/pom.xml"
+    "../extensions/tekton/pom.xml"
+    "../extensions/tekton/pom.xml"
+    "../extensions/volumesnapshot/pom.xml"
+)
+
+declare -a extensionModules=(
+    "../extensions/knative/generator"
+    "../extensions/camel-k/generator-v1"
+    "../extensions/camel-k/generator-v1alpha1"
+    "../extensions/chaosmesh/generator"
+    "../extensions/service-catalog/generator"
+    "../extensions/tekton/generator-v1alpha1"
+    "../extensions/tekton/generator-v1beta1"
+    "../extensions/volumesnapshot/generator"
+)
```

Comment:
I'd rather keep things simpler. Why not declare those together with the rest of modules?